cocos/render-scene/scene/skybox.ts

```diff
@@ -318,7 +318,7 @@ export class Skybox {
         this._default = builtinResMgr.get<TextureCube>('default-cube-texture');
 
         if (!this._model) {
-            this._model = cclegacy.director.root.createModel(cclegacy.renderer.scene.Model) as Model;
+            this._model = (cclegacy.director.root as Root).createModel(cclegacy.renderer.scene.Model as typeof Model);
```

Comment:
Why can not use `as Model`?

Reply:
Because `instance as Model` used for instance means converting `instance` to `Model` type.
But `constructor as typeof Model` is used for converting `a constructor`.

You could refer to the error message reported by tsc:
```
Argument of type 'Model' is not assignable to parameter of type 'typeof Model'.
  Property 'prototype' is missing in type 'Model' but required in type 'typeof Model'.ts(2345)
```